scripts/wdl/pon_gatk_workflow.wdl

```diff
@@ -0,0 +1,358 @@
+    Int padding = 250
+    File gatk_jar
+    Boolean isWGS
+    Int mem
+
+    # Note that when isWGS is true, this task is still called by the workflow.
+    # In that case, to satisfy the output requirements an empty target file is created, that gets 
+    # passed to the CalculateTargetCoverage task to satisfy its input requirements.
+    # Regardless of isWGS value, the output of PadTargets is passed to WholeGenomeCoverage which
+    # then outputs the target file accordingly(see below)
+    command {
+        if [ ${isWGS} = false ]; \
+          then java -Xmx${mem}g -jar ${gatk_jar} PadTargets --targets ${target_bed} --output targets.padded.tsv \
+                --padding ${padding} --help false --version false --verbosity INFO --QUIET false; \
+          else touch targets.padded.tsv; \
+        fi
+    }
+
+    output {
+        File padded_target_bed = "targets.padded.tsv"
+    }
+
+    #runtime {
+    #    docker: "gatk-protected/a1"
+    #}
+}
+
+# Helper task to get the name of the given bam file
+task GetBamFileName {
+    File input_bam
+
+    command <<<
+        echo $(basename "${input_bam}" .bam)
+     >>>
+
+    output {
+        String name=read_string(stdout())
+    }
+}
+
+# Calculate the target proportional coverage
+task CalculateTargetCoverage {
+    String entity_id
+    File padded_target_bed
+    String transform = "PCOV"
+    String grouping = "SAMPLE"
+    Boolean keep_duplicate_reads = true
+    Boolean disable_all_read_filters = false
+    File input_bam
+    File bam_idx
+    File ref_fasta
+    File ref_fasta_fai
+    File ref_fasta_dict
+    File gatk_jar
+    String is_disable_reference_validation
+    Boolean isWGS
+    Int mem
+
+    # Note that when isWGS is true, this task is still called by the workflow.
+    # In that case, to satisfy the output requirements an empty coverage file is created
+    # and passed to the WholeGenomeCoverage task.
+    command {
+        if [ ${isWGS} = false ]; \
+          then java -Xmx${mem}g -jar ${gatk_jar} CalculateTargetCoverage --output ${entity_id}.coverage.tsv \
+                --groupBy ${grouping} --transform ${transform} --targets ${padded_target_bed} --targetInformationColumns FULL \
+                --keepduplicatereads ${keep_duplicate_reads} --input ${input_bam} --reference ${ref_fasta} \
+                --disable_all_read_filters ${disable_all_read_filters} --interval_set_rule UNION --interval_padding 0 \
+                --secondsBetweenProgressUpdates 10.0 --disableSequenceDictionaryValidation ${is_disable_reference_validation} \
+                --createOutputBamIndex true --help false --version false --verbosity INFO --QUIET false; \
+          else touch ${entity_id}.coverage.tsv; \
+        fi
+    }
+
+    output {
+        File gatk_coverage_file = "${entity_id}.coverage.tsv"
+    }
+
+    #runtime {
+    #    docker: "gatk-protected/a1"
+    #}
+}
+
+# Caclulate coverage on Whole Genome Sequence using Spark. This task automatically creates
```

Comment:
Caclulate -> Calculate.  Also, line break placement here is different from same text above.